.changeset/unlucky-beans-obey.md

```diff
@@ -0,0 +1,5 @@
+---
+'openzeppelin-solidity': patch
+---
+
+`ERC2771Context`: Return the forwarder address whenever `msg.data.length` is not appended with the request signer address (i.e. length is less than 20 bytes).
```

Comment:
I'm not sure what "`msg.data.length` is not appended with" means. The address is appended to te msg.data, and the msg.data.length increases by 20.

```suggestion
`ERC2771Context`: Return the forwarder address whenever the `msg.data` or calls sent by a trusted forwarder is not long enough to contain the request signer address (i.e. `msg.data.length` is less than 20 bytes).
```